.editorconfig

```diff
@@ -15,5 +15,8 @@ indent_size = 2
 [*.md]
 indent_size = 2
 
+[*.java]
+indent_size = 2
+
```

Comment:
Could you remove this? This isn't relevant here, and such things are covered by the shared style guide in build-tools. You can read more [here](https://github.com/zeebe-io/zeebe/wiki/Code-Style)